instrumentation/apache-camel-2.20/javaagent/build.gradle.kts

```diff
@@ -64,5 +64,8 @@ tasks {
     // TODO run tests both with and without experimental span attributes
     jvmArgs("-Dotel.instrumentation.apache-camel.experimental-span-attributes=true")
     jvmArgs("-Dotel.instrumentation.aws-sdk.experimental-span-attributes=true")
+
+    // TODO: fix camel instrumentation so that it uses semantic attributes extractors
+    jvmArgs("-Dotel.instrumentation.experimental.span-suppression-strategy=span-kind")
```

Comment:
Camel instrumentation does not use any of our semantic convention specific utilities - fixing it would pretty much require rewriting most of it, that's too big to be included in this PR.